about/changelog.md

```diff
@@ -8,6 +8,83 @@ keywords: [changelog, upgrades, updates, releases]
 
 All the latest features and updates to Timescale products.
 
+
+## 🏄 Optimizations for compressed data and extended join support in continuous aggregates
+<Label type="date">July 31, 2024</Label>
```

Comment:
We announce when the release is available to all customers in production. So. assuming that no blockers are found and 2.16 gets into production in Timescale Cloud next week:

```suggestion
<Label type="date">August 15, 2024</Label>
```